src/locationService.ts

```diff
@@ -124,14 +122,11 @@ export class LocationsClient {
       opts['scopes'] = staticMembers.scopes;
     }
 
-    // Create a `gaxGrpc` object, with any grpc-specific options sent to the client.
-    this._gaxGrpc = new GrpcClient(opts);
-
     // Save options to use in initialize() method.
     this._opts = opts;
 
     // Save the auth object to the client, for use by other methods.
-    this.auth = this._gaxGrpc.auth as GoogleAuth;
+    this.auth = gaxGrpc.auth as GoogleAuth;
```

Comment:
Why are you removing gaxGrpc from the instantiated class?

Reply:
This is one of the reasons why the existing code does not work in browser with webpack.

To ask the client class to be browser-compatible and use `fallback.ts` implementation, we pass `fallback: 'proto'` or `fallback: 'rest'` to the constructor, so that the client won't use gRPC (which does not work in browser) and use proto-over-HTTP or REGAPIC (JSON-over-HTTP) instead.

```ts
const client = new library.WhateverClient({fallback: 'proto'}); // fallback: true also works
```

This affects how the client accesses `gax` (whether it would choose `gax` or `gax.fallback`), and what it will assign to its `gaxGrpc`, e.g. [here](https://github.com/googleapis/nodejs-speech/blob/b6c4c7ecd013c66446028fe2c48b400cb0f8c3b2/src/v1/speech_client.ts#L122-L126) is this place in Speech API.

Now, the Location service is a "mixin", basically, it's a separate client class that can be attached to the client that exposes Location API (that's why it lives in the common place, in `google-gax`).

If Speech client used `fallback` and created `gaxGrpc` from the fallback implementation, it must use the same `gaxGrpc` for all other clients (Location and LRO). For that to happen, the constructors for Location client and Operations client accept the existing instance of `gaxGrpc` (line 101 of this file). But I found that the instance that was passed is not actually used in some of these places, which makes the fallback client instantiate Location client that uses gRPC, so I'm changing all these places to use the instance that was passed down.